plugins/callback/foreman.py

```diff
@@ -19,6 +19,15 @@
       - whitelisting in configuration
       - requests (python library)
     options:
+      report_type:
```

Comment:
I wonder if this is even needed, given you need to configure a different URL.

Shouldn't "proxy_url is configured, send proxy format, otherwise send foreman format" be sufficient?